plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -1063,16 +1063,47 @@ public void testAlterTableWithUnsupportedProperties()
 
         assertUpdate("CREATE TABLE " + tableName + " (a_number INT)");
 
-        assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true",
-                "The following properties cannot be updated: change_data_feed_enabled");
         assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = true, checkpoint_interval = 10",
-                "The following properties cannot be updated: change_data_feed_enabled, checkpoint_interval");
+                "The following properties cannot be updated: checkpoint_interval");
         assertQueryFails("ALTER TABLE " + tableName + " SET PROPERTIES writer_version = 4, partitioned_by = ARRAY['a']",
                 "The following properties cannot be updated: partitioned_by");
 
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testSettingChangeDataFeedEnabledProperty()
+    {
+        String tableName1 = "test_enable_and_disable_cdf_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tableName1 + " (page_url VARCHAR, domain VARCHAR, views INTEGER)");
+        assertThat((String) computeScalar("SHOW CREATE TABLE " + tableName1)).contains("writer_version = 2");
+
```

Comment:
we can include test for `ALTER TABLE " + tableName + " SET PROPERTIES change_data_feed_enabled = false, writer_version = 3`